src/Sylius/Bundle/ApiBundle/Controller/DeleteOrderItemAction.php

```diff
@@ -37,6 +37,6 @@ public function __invoke(Request $request): Response
 
         $this->commandBus->dispatch($command);
 
-        return new JsonResponse();
+        return new JsonResponse(null, Response::HTTP_NO_CONTENT);
```

Comment:
@lchrusciel Api platform returns this kind of response for DELETE endpoints instead of returning 200 with {} in content. I noticed this while adding contract test. Let me know if you are ok with it